storage.go

```diff
@@ -211,6 +221,12 @@ func (s *Storage) stat(ctx context.Context, path string, opt pairStorageStat) (o
 // First we need make sure this file is not exist.
 // If it is, then we upload it, or we will overwrite it.
 func (s *Storage) write(ctx context.Context, path string, r io.Reader, size int64, opt pairStorageWrite) (n int64, err error) {
+
+	err = makeCache()
```

Comment:
If we init the cache while `NewStorager`, we can avoid the usage for `sync.Once`.

Reply:
Looks very simple and elegant solution! I'll do this. Thanks a lot.